test/schema/export_test.js

```diff
@@ -21,14 +24,27 @@ describe('#exportToSchemasUniqueCases()', () => {
     let schemas = exportToSchemas([ns]);
     expect(schemas).to.have.length(16);
     expect(schemas).to.eql(expectedSchemas);
+    expect(validateSchema(schemas));
```

Comment:
OK.  If you confirmed it does what it should then that's OK with me.  But I'm still not sure how it works at all since the `validate` function arguments don't seem to be correct...